tester/main.cpp

```diff
@@ -34,34 +37,22 @@ typedef struct cat_t {
 } category_t;
 
 int main(int argc, char const *argv[]) {
-	// common options
-	vc::Option includeCustomUtils("custom_include", "-include", "utils.hpp");
-	vc::Option includeCustomUtilsDir("custom_includeDir", "-I", "../include");
-	vc::Option includePolybenchUtils("polybench_include", "-include", "polybench.c");
-	vc::Option includePolybenchUtilsDir("pb_Dir", "-I", polybench_source + "/utilities");
-	vc::Option optimizationLevel("O","-O","3");
-	vc::Option standardCxx("std","-std=","c++11");
-	std::list<vc::Option> default_options = {
-		standardCxx,
-		includePolybenchUtils,
-		includeCustomUtils,
-		optimizationLevel,
-		includePolybenchUtilsDir,
-		includeCustomUtilsDir,
-	};
```

Comment:
some of these default options may not be needed anymore, however I still believe the common options should be defined outside loop as they are loop-invariant.